
# Use udevrule for piksi:
# SUBSYSTEMS=="usb", KERNEL=="ttyUSB[0-9]*", ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6014", SYMLINK+="piksi", MODE="0666", GROUP="dialout"
serial_port: '/dev/piksi'
baud_rate: 1000000
base_station_mode: false
# whenever possible, use network broadcast, not global ip broadcast
# use ifconfig to show network broadcast (or calculate it yourself....)
broadcast_addr: 10.10.50.255
broadcast_port: 26078
base_station_ip_for_latency_estimation: 10.10.50.1 # ip base station if you want to estimate latency
# Coordinate frame of the GPS receiver with respect to the base link
frame_id: 'base_link'

# Covariances to be published in the NavSatFix message
# Single Point Positioning (SPP)
var_spp: [33.0, 25.0, 64.0]
# RTK float mode
var_rtk_float: [33.0, 25.0, 64.0]
# RTK fix mode
var_rtk_fix: [0.0049, 0.0049, 0.01]
